frontend-react/src/components/FileHandlers/FileHandler.test.tsx

```diff
@@ -181,6 +184,7 @@ describe("FileHandler", () => {
                     showSuccessMetadata={false}
                     showWarningBanner={false}
                     warningText=""
+                    endpointName="waters"
```

Comment:
can we turn this into an enum? "waters" and "validate"?

Reply:
done!